graylog2-web-interface/src/views/components/visualizations/XYPlot.jsx

```diff
@@ -38,7 +38,7 @@ const XYPlot = ({
   chartData,
   currentQuery,
   timezone,
-  effectiveTimerange,
+  effectiveTimerange = {},
```

Comment:
With the migration we are adding the new widgets to the current view and fetching the search result afterwards.
Because the widgets are visible, before the search got executed, they are throwing a few errors, because of the missing result data object. I thought about passing a function to the `MigrateFieldCharts` component to hide all widgets before finishing the search execution. Another solution is to define the related attributes as optional in the `XYPlot.jsx` an `TransformKeys.js`. Which way do you prefer?

Reply:
Answer: I will try to show the loading spinner for the `SearchResult` component, if it's not a suitable solution, I will try the second way.